### DISCUSSION TITLE
School’s Out for 2023.. Learning, Laughing, and Looking Ahead to 2024! [Check-in]

### DISCUSSION THREAD

**Discussion by samus-aran** [community/community#81362] [GitHub Education]
As we gather here for our final Education check-in of the year, we can't help but feel a profound sense of gratitude for each and every one of you. As we bid farewell to this remarkable year, let's take a moment to acknowledge the collective achievements of our community.

## 🏆​ 2023’s Top Contributors in the Education Category  

<img src="https://github.com/community/community/assets/4041275/ba66a04d-ddfc-4e76-a1d0-4417336d4e1a" width="75%"/>

> Alt text: [Three astronauts on an Olympic style podium depicting first, second, and third place against a space background. The astronauts' faces are the profile pictures of the top contributors.]

- @arafajs
- @mantasu
- @GhostOf0days

## Education Award Winners @ GitHub Universe

At this year’s GitHub Universe we were proud to present GitHub Awards to some of our [Education community members](https://github.blog/2023-11-09-celebrating-the-github-awards-2023-recipients/#education-awards) in person!

- [Lighthouse Award](https://github.blog/2023-11-09-celebrating-the-github-awards-2023-recipients/#lighthouse-award) - @GhostOf0days
- [Empowering Educator Award](https://github.blog/2023-11-09-celebrating-the-github-awards-2023-recipients/#empowering-educator-award) - @catherine-leung
- [Phenomenal Education Partner Award](https://github.blog/2023-11-09-celebrating-the-github-awards-2023-recipients/#phenomenal-education-partner-award) - Appwrite

Thank you members, for making this community a truly exceptional place to learn and grow. Here's to a year of triumphs, and to the countless possibilities that await us in the coming year!

Everyone deserves a round of applause! 👏👏👏

### Advent of Code 📅 💻

You still have a few remaining days to try out [Advent of Code](https://adventofcode.com/) and get into the festive spirit 🎄

Advent of Code is an annual online event where participants solve a series of programming puzzles or challenges. The challenges are released daily where participants are tasked with solving it using programming skills until December 25th.

To participate in Advent of Code, you can visit the [website](https://adventofcode.com/) during the event period.

Share with the community if you were ranked in the [Leaderboard](https://adventofcode.com/2023/leaderboard) 👀

### MLH Beginners Week

As part of [Major League Hackers’s](https://ghw.mlh.io/) monthly Global Hack Week, January 5th - 11th, 2024 will be [Beginners Week](https://ghw.mlh.io/events/beginners-week2)! Join them for a week of beginner-friendly content to kick off 2024! 🎇

Global Hack Week is a 100% free event for anyone and it's easy to register and participate - follow along with [live challenges](https://ghw.mlh.io/challenges). 👍

### Newest Experience: Security & Monitoring

This new GitHub Education Experience will allow you to explore industry standard tools that keep your code safe, your permissions secure and your infrastructure monitored so that you can confidently scale your brilliant ideas. 

Discover how Dependabot and Advanced Security features work at Github and explore tutorials to set up code scanning along with offers included in the [Student Developer Pack](https://education.github.com/pack): Astra, Doppler, 1Password, Honeybadger and more.

Checkout others awesome GitHub Education Experiences in our [Benefits page](https://education.github.com/pack).

### Upcoming Changes to Assignment Repositories and Starter Code in GitHub Classroom

New Year is off to an exciting start with important changes to Classroom so keep these updates on your calendar in January. 

Currently, starter code repositories must be template repositories, and GitHub Classroom [creates a repository from a template](https://docs.github.com/repositories/creating-and-managing-repositories/creating-a-repository-from-a-template) for each student repository. After the change, student repositories will be created by [forking](https://docs.github.com/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks) the starter code repository.

It was thanks to your feedback from the Educators community that this change will now grant the ability to change starter code after an assignment has been accepted by students. 🎉Check out what else is planned for the January update in the [Changelog blog post](https://github.blog/changelog/2023-12-08-upcoming-changes-to-assignment-repositories-and-starter-code-in-github-classroom/).

## To the future

Here's to a fantastic year behind us and an even more remarkable one ahead. Keep pushing boundaries, sharing knowledge, and supporting one another. The community's strength lies in its diversity and collective brilliance.

Wishing you all a joyful festive season, a restful break, and a thrilling start to the upcoming year. Keep it awesome! ⭐

**Comment by dgilleland** [comment]
@samus-aran + @Iron-Ham 
Wondering when the January update is coming into effect. It's currently **Jan 3**, 2024, and many of us have classes starting as early as tomorrow. Our prep time is limited, so getting some "hands-on" to see the effects of the changes is pretty important to us!

**Reply by SirYancy** [reply]
I'd like to second this question. I'm looking at this class starting Monday and I'd like to get my labs uploaded.

**Reply by grady** [reply]
I too would like to know when the template/fork changes are going live. Spring classes are starting very soon for a lot of us and we need to get the new workflows nailed down ASAP. 

If it is going to be after Monday the 6th, you need to let us know that things are going to be changing mid-term for a lot of us.

**Reply by dgilleland** [reply]
Because GitHub Classroom is used with teaching, issues like this are **time-sensitive**. Can @Iron-Ham or @samus-aran give feedback on this?

**Reply by doctorscott** [reply]
Agreed! When is this happening? It was announced on Dec 8th without a timeline. Not very professional! We really need to know please!

**Reply by ron-reaser** [reply]
I also need to know when this change is occurring. I need to release my term project repository by Friday 1/19. I only get one shot for the entire spring semester, and I would prefer to do so under the new system to avoid merge conflicts.
